### PR TITLE
Create project in tests

### PR DESCRIPTION
**Pull request by harminius**
Minifix to send ProjectVersion.created signal when creating project in tests.

On this signal, a Workspace Contributor entry is created. It's useful in tests.

### CODE REVIEW

File: server/mergin/tests/utils.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def create_project(name, workspace, user, **kwargs):
     pv = ProjectVersion(p, 0, user.id, changes, "127.0.0.1")
     db.session.add(pv)
     db.session.commit()
+    project_version_created.send(pv)
```
Comment (varmar05) [inline server/mergin/tests/utils.py:91]: Should not we trigger signal from ProjectVersion constructor instead?

Comment (harminius) [reply server/mergin/tests/utils.py:91]: That would be probably the safest.

Comment (MarcelGeo) [reply server/mergin/tests/utils.py:91]: Yes

Comment (varmar05) [reply server/mergin/tests/utils.py:91]: Indeed, I suggest some refactoring here, best to set it in one place and remove elsewhere.

**Review by MarcelGeo — APPROVED** [review]

**Pull request merged**